scripts/run_backend_tests.py

```diff
@@ -414,6 +419,8 @@ def check_test_results(
 def main(args: Optional[List[str]] = None) -> None:
     """Run the tests."""
     parsed_args = _PARSER.parse_args(args=args)
+    if not parsed_args.skip_install:
+        install_third_party_libs.main()
```

Comment:
@FaithAbiola The issue states that we should skip installing third_party libraries if the `--skip-install` flag is passed. So, shouldn't we add a conditional check where this function is called instead of creating a new new call?